spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/server/Cookie.java

```diff
@@ -145,6 +145,11 @@ public void setPartitioned(Boolean partitioned) {
 	 */
 	public enum SameSite {
 
+		/**
+		 * Don't set the SameSite cookie attribute.
+		 */
+		UNSET("Unset"),
```

Comment:
It may just be me, but `UNSET` makes me have to stop and think to understand what it does. I wonder if `OMITTED`, described as "The SameSite cookie attribute is omitted" would be clearer?

Also, could the attribute value be `null`? I wonder if that would simplify the logic for applying it.

Reply:
I remember now why I picked `UNSET`, it's because the enum `org.apache.tomcat.util.http.SameSiteCookies` has it. In any case `OMITTED` is more clear so I'll go ahead and use that